.travis/deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+mvn deploy -DskipTests -P release --settings cd/mvnsettings.xml
```

Comment:
the mvnsettings has been added in .travis folder instead of cd folder

Reply:
What is your reasoning to use `cd` as a directory name? is that a standard across some projects? IMO, when used outside the CI/CD it loses its identity. It was confusing with `cd` shell command, thus I choose to relate with `.travis` directory.

Reply:
not aware of specific standard.
@lhein do you remember if you picked it from a standard or you created it?

My point was just to point that the argument is using "cd" although the fiel has been added to `.travis`. I don't know the implication of putting it in `.travis` folder.

Reply:
It was taken from an example script I found somewhere.

Reply:
The `.travis` folder should not affect the Travis build, and being in the `.travis` directory it can be hidden from day to day users easily, I did take from the cue from one of the examples of GPG blogs. The Travis docs show inlining everything in the .travis.yml file itself too which may remove this confusion. I will be open to that.

Reply:
ok, seems a good idea to move to .travis folder then.
the parameter on the pointed line needs to be updated accordingly

Reply:
fixed the cd name to .travis